services/centralized-kubecost/0.32.0/defaults/cm.yaml

```diff
@@ -2,46 +2,11 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: centralized-kubecost-0.31.0-d2iq-defaults
+  name: centralized-kubecost-0.32.0-d2iq-defaults
   namespace: ${releaseNamespace}
 data:
   values.yaml: |
     ---
-    opencost:
-      enabled: true
-      securityContext:
-        allowPrivilegeEscalation: false
-        runAsUser: 1001
-        runAsNonRoot: true
-      resources:
-        limits:
-          cpu: 999m
-          memory: 1G
-        requests:
-          cpu: 10m
-          memory: 55M
-      env:
-        - name: PROMETHEUS_SERVER_ENDPOINT
-          valueFrom:
-           configMapKeyRef:
-             key: prometheus-server-endpoint
-             name: kommander-kubecost-cost-analyzer
-        - name: THANOS_ENABLED
-          value: "true"
-        - name: THANOS_QUERY_URL
-          value: http://kommander-kubecost-thanos-query-http.kubecost.svc.cluster.local:10902
-        - name: THANOS_QUERY_OFFSET
-          value: 5m
-        - name: THANOS_MAX_SOURCE_RESOLUTION
-          value: 5m
-        - name: CLOUD_PROVIDER_API_KEY
-          value: "AIzaSyD29bGxmHAVEOBYtgd8sYM2gM2ekfxQX4U" # The GCP Pricing API requires a key. This is supplied just for evaluation.
-        - name: CLUSTER_ID
-          valueFrom:
-           configMapKeyRef:
-             key: CLUSTER_ID
-             name: kubecost-cluster-info-configmap
-
```

Comment:
Were these lines only relevant for v0.31.0?

Reply:
yep in `0.32.0` we removed some of the config related to opencost, so we can remove it from here as well